.changeset/short-rabbits-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
+"@espressive/cascara": patch
+---
+
+feat(DEV-373):  [Cascara] - Columns component mvp
```

Comment:
@lordpixel I think you did the component that was being added before in brackets, right? That seems better to me so we can see all the different component related things together in the release notes.

I am also creating a separate task before I forget that we need to get the latest release changelog copied to the package README each time so when you look at the releases in Github you can see everything that was included in that release. That is not part of this ticket... your changeset just reminded me.

Reply:
You are right, I used this Cascara scope because Cascara was the package the Columns component was added to. I agree with you that this should be the component name, which makes me think: Could we use some kind of namespace? 

i.e. feat(FDS-373): [Cascara - Columns] -  Columns component mvp

The idea is to have the name of the package like `Cascara`, `Tokens`, `Icons` etc and then the name of the piece of code that was changed. Let me update the changeset 👍